doc/api/packages.md

```diff
@@ -69,13 +69,14 @@ expressions:
 * Strings passed in as an argument to `--eval`, or piped to `node` via `STDIN`,
   with the flag `--input-type=module`.
 
-* Code that contains syntax that only parses successfully as [ES modules][],
+* When using [--experimental-detect-module](https://nodejs.org/docs/latest/api/cli.html),
```

Comment:
Like other links, it needs to be a reference you add at the end of the document.
```suggestion
* When using [`--experimental-detect-module`][]
```

Reply:
I didn't know that. Thanks.